tests/utils/couchdb-utils.js

```diff
@@ -59,17 +59,37 @@ export const contacts = () => docs.filter(doc => contactTypes.includes(doc.type)
 
 const getDbByDoc = (id) => Object.keys(docsByDb).filter(dnName => docsByDb[dnName].has(id));
 
-export const editDoc = async (doc) => {
+export const insertDocs = async (documents) => {
+  docs.push(...documents);
+  const db = getDb(dbNames[0]);
+  await db.bulkDocs(documents);
+};
+export const editDoc = async (doc, useOldRev = false) => {
   const dbName = getDbByDoc(doc._id);
   const db = getDb(dbName);
-  const existentDoc = await db.get(doc._id);
-  await db.put({ ...doc, _rev: existentDoc._rev });
+  let existentDoc;
+  if (useOldRev) {
+    existentDoc = await db.get(doc._id);
+    doc._rev = existentDoc._rev;
+  } else {
+    existentDoc = await db.get(doc._id);
+    doc._rev = existentDoc._rev;
+  }
+
+  await db.put(doc, { new_edits: true });
```

Comment:
`PouchDb.put` doesn't have a `new_edits` option: https://pouchdb.com/api.html#create_document
I think only `bulkDocs` allows the `new_edits` option: https://pouchdb.com/api.html#batch_create

Or at least this is how it's documented. 
Once you fix the code so you don't use the old rev, then it should become evident if the `put` fails when you omit the rev.

Reply:
Yes, you are right I'm getting:
` error at async HttpPouch.api._put (node_modules/pouchdb-adapter-http/lib/index.js:769:22)`
I'm changing this to use bulkDocs.